src/components/Sections/VideoSection/index.tsx

```diff
@@ -0,0 +1,54 @@
+import BaseComponent from "@/components/BaseComponent";
+import LineSeparator from "@/components/LineSeparator";
+import Headline from "@/components/Headline";
+import { VideoSectionProps } from "@/types/sections";
+import { Component, Prop } from "vue-property-decorator";
+
+@Component({
+  name: "VideoSection",
+})
+class VideoSection extends BaseComponent<VideoSectionProps> {
+  @Prop() youtubeId!: VideoSectionProps["youtubeId"];
+  @Prop() width: VideoSectionProps["width"];
+  @Prop() height: VideoSectionProps["height"];
+  @Prop() title: VideoSectionProps["title"];
+  @Prop() description: VideoSectionProps["description"];
+  @Prop() parameters: VideoSectionProps["parameters"];
+  @Prop() youtubeUrl: VideoSectionProps["youtubeUrl"];
+
+  properties() {
+    return this.parameters
+      ? this.parameters
+          .map((p, index) => (index > 0 ? "&" : "?") + p.param + "=" + p.value)
+          .join("")
+      : "";
```

Comment:
We should encode all URL parameters:
```suggestion
    return this.parameters
      ? this.parameters.reduce((resultStr, item, index) => `${resultStr}${index > 0 ? '&' : ''}${item.param}=${encodeURIComponent(item.value)}`, '?')
      : "";
```

Reply:
Looking at the [supported parameters](https://developers.google.com/youtube/player_parameters#Parameters) I don't think we need encoding here: most of the possible values are "0" or "1", and the rest is well-defined without too (without any problematic characters). Not sure if I prefer `reduce` over `map` and `join`... but ok.